lib/osf-components/addon/components/provider-metadata-editable/component.ts

```diff
@@ -0,0 +1,11 @@
+import Component from '@ember/component';
```

Comment:
Maybe also consider a glimmer component here?

Reply:
As above, this failed to render when I tried converting it, and I'm not really sure why.